tools/p11ne-db

```diff
@@ -65,8 +65,18 @@ cmd_pack-key() {
     which aws > /dev/null
     ok_or_die "aws cli is not present on this machine. Please install the aws cli and try again."
 
+     # Fetch AWS CLI version
+    local aws_cli_ver
+    aws_cli_ver=$(aws --version 2>&1 | cut -d " " -f1 | cut -d "/" -f2 | cut -d "." -f1)
```

Comment:
Is the redirect of stderr to stdout necessary?

Reply:
yes, this is to match the aws cli v1 printing to stderr: https://stackoverflow.com/questions/43280287/why-aws-version-writes-to-stderr